contracts/DaiHrd.sol

```diff
@@ -569,28 +562,26 @@ contract DaiHrd is ERC777, MakerFunctions {
 	}
 
 	// Utility Functions
-	function convertAttodaiToAttodaiHrd(uint256 attodai, uint256 rontodaiPerPot ) internal pure returns (uint256 attodaiHrd) {
+	function calculatedChi() public view returns (uint256 rontodaiPerPot) {
+		// mirrors Maker's calculation: rmul(rpow(dsr, now - rho, ONE), chi);
+		return safeMul27(rpow(pot.dsr(), now - pot.rho(), 10 ** 27), pot.chi());
+	}
+
+	function convertAttodaiToAttodaiHrd(uint256 attodai, uint256 rontodaiPerPot ) private pure returns (uint256 attodaiHrd) {
 		// + 1 is to compensate rounding? since attodaiHrd is rounded down
 		return attodai.mul(10 ** 27).add(rontodaiPerPot - 1).div(rontodaiPerPot);
 	}
 
-	function convertAttodaiHrdToAttodai(uint256 attodaiHrd, uint256 rontodaiPerPot ) internal pure returns (uint256 attodai) {
+	function convertAttodaiHrdToAttodai(uint256 attodaiHrd, uint256 rontodaiPerPot ) private pure returns (uint256 attodai) {
 		return attodaiHrd.mul(rontodaiPerPot).div(10 ** 27);
 	}
 
-	function calculatedChi() public view returns (uint256 rontodaiPerPot) {
-		return safeMul27(rpow(pot.dsr(), now - pot.rho(), ONE), pot.chi());
-	}
-
-	function updateAndFetchChi() internal returns (uint256 rontodaiPerPot) {
-		if (pot.rho() == now) {
-			return pot.chi(); // replace with drop()?? TODO
-		}
-		return pot.drip();
+	function updateAndFetchChi() private returns (uint256 rontodaiPerPot) {
+		return (pot.rho() == now) ? pot.chi() : pot.drip();
```

Comment:
Switched to a ternary to make it a little more clear that this function returns either `pot.chi()` or `pot.drip()` at a glance.  Also removed the TODO since Maker isn't launching with the proposed `drop`.